src/Tester/StreamingTests/PullingAgentManagementTests.cs

```diff
@@ -16,30 +16,23 @@ namespace UnitTests.StreamingTests
     [DeploymentItem("ClientConfigurationForStreamTesting.xml")]
     [DeploymentItem("OrleansProviders.dll")]
     [TestClass]
-    public class PullingAgentManagementTests : UnitTestSiloHost
+    public class PullingAgentManagementTests : UnitTestSiloHostPerFixture
```

Comment:
well, today it would map to a class fixture. Although after the migration I intend to map it to the broader collection fixture, and with that have even less restarts.